examples/cryptocurrency-advanced/backend/build.rs

```diff
@@ -0,0 +1,13 @@
+extern crate exonum_build;
+
+use exonum_build::protobuf_generate;
+use std::env;
+
+fn main() {
+    let exonum_protos = env::var("DEP_EXONUM_PROTOBUF_PROTOS").unwrap();
```

Comment:
Hm... maybe we should consider making this a part of `exonum_build::protobuf_generate`. It's highly likely that _all_ Exonum services will need Exonum Protobuf definitions.

Reply:
This one I agree, I was thinking to do that since we hide exact implementation of the way we pass information about .proto file names.